docs/latest/modules/en/pages/k8s-suse-rancher-prime.adoc

```diff
@@ -34,50 +34,63 @@ The Non-HA setup can support up to 100 Nodes under observation.
 |  | trial | 10 non-HA | 20 non-HA | 50 non-HA | 100 non-HA | 150 HA | 250 HA | 500 HA | 4000 HA
 
 | *CPU Requests*
-| 7.5
-| 7.5
-| 10.5
-| 15
-| 25
-| 49
-| 62
-| 86.5
-| 210
+| 6945m
```

Comment:
i think we should round these numbers up like it was before, (half cores?) this is hard to read i think